docs: update guide for chat memory (#4703)
langflow-ai/langflow: docs/docs/Guides/guides-chat-memory.md

Modifying these memories will influence the behavior of the chatbot responses, as long as an agent uses them. Here you have the ability to remove or edit previous messages, allowing them to manipulate and explore how these changes affect model responses.

> To modify chat memories, open the playground, click on the menu of any session on the right, and then click on the **Message Logs** option.

Reviewer: ```suggestion
To modify chat memories, from the playground, click the **Options** menu of any session, and then select **Message Logs**
```

![](/img/chat-input-controls-pane.png)

> By default, if the `Session ID`  value is empty, it is set to match the the same value as the `Flow ID`.

Reviewer: ```suggestion
By default, if the `Session ID` value is empty, it is set to the same value as `Flow ID`.
```
